docs/Components/BaseComponent.md

```diff
@@ -98,6 +98,18 @@ base_component.add_external_force_function('force1', force1)
 print(base_component.external_forces)
 >>> {'force1': <function force1 at 0x0000025EBCAA8D30>}
 ```
+## *add_external_moment_function*:
+This method is responsible for appending an external moment function to the component and it should return the magnitude of the moment only.
```

Comment:
You got it.
Just to make it clearer:

This method is responsible for appending an external moment function to the component. The moment function should return a float representing the magnitude of the moment.